common/helpers/Cargo.toml

```diff
@@ -17,6 +17,8 @@ apache-avro = { git = "https://github.com/apache/avro", features = ["snappy"] }
 thiserror = "1"
 sha2 = "0.10.2"
 rayon = "1.5"
+# serde_json_core = { version = "0.3" , default-features = false }
+serde_json = { version = "1.0", default-features = false }
```

Comment:
shouldn't this be the following based on https://crates.io/crates/serde_json#no-std-support?

`serde_json = { version = "1.0", default-features = false, features = ["alloc"] }`

Reply:
Yes, so I'll create another pr for using serde-json-core. currently that library is failing to compile